csrc/scheduler/hopper_multi_matmul.cpp

```diff
@@ -1332,6 +1396,8 @@ void HopperMultipleMatmulScheduler::setUpCircularBuffering() {
     }
   }
 
+  /*
+  // TODO Investigate. Disable loop rotation with tma circular buffering
```

Comment:
My first thought was can we just disable this parameter for loop rotation, but I realized that we do not actually respect the `rotate_ldmatrix_out_of_main_loop` parameter in `MatmulParams`. In fact, I just ran `git log --patch csrc/scheduler/ | grep -C20 rotate_` and sifted through the results. I don't think we've **ever** used that parameter :-D cc @zasdfgbnm .